docker/otel-collector/config.yaml

```diff
@@ -12,18 +12,19 @@ receivers:
   # Data sources: logs
   fluentforward:
     endpoint: '0.0.0.0:24225'
+  # Configured via OpAMP w/ authentication
   # Data sources: traces, metrics, logs
-  otlp:
-    protocols:
-      grpc:
-        include_metadata: true
-        endpoint: '0.0.0.0:4317'
-      http:
-        cors:
-          allowed_origins: ['*']
-          allowed_headers: ['*']
-        include_metadata: true
-        endpoint: '0.0.0.0:4318'
+  # otlp/hyperdx:
```

Comment:
moved to `opampController`, this is so we don't enable these receivers until we know for sure which authentication mode we want